pkg/tsdbctl/create.go

```diff
@@ -212,3 +248,10 @@ func rateToHours(sampleRate string) (int, error) {
 		return 0, parsingError
 	}
 }
+
+func getNextNumberEvenlyDivided(n, d1, d2 int) int {
+	var i int
+	for i = n; i%d1 != 0 || i%d2 != 0; i-- {
+	}
```

Comment:
Why are we confident that this will always yield a desirable result? It seems rather brute force, with potential to run forever on certain inputs.